Source/FieldSolver/WarpXExternalEMFields.cpp

```diff
@@ -154,10 +161,17 @@ WarpX::ApplyExternalFieldExcitationOnGrid (
                 WarpXUtilAlgo::getCellCoordinates(i, j, k, mfx_stag,
                                                   problo, dx, x, y, z);
                 auto flag_type = xflag_parser(x,y,z);
+                amrex::dt_type_factor = 1._rt;
+                // For soft source and FirstHalf/SecondHalf evolve
+                // the excitation is split with a prefector of 0.5
+                if (flag_type == 2._rt and dt_flag_type == 1) {
```

Comment:
all instances of `dt_flag_type` should be `dt_type_flag`

Reply:
thanks Andy. fixed!